library/x509_crt.c

```diff
@@ -167,6 +167,9 @@ const mbedtls_x509_crt_profile mbedtls_x509_crt_profile_suiteb =
 static int x509_profile_check_md_alg( const mbedtls_x509_crt_profile *profile,
                                       mbedtls_md_type_t md_alg )
 {
+    if( md_alg == MBEDTLS_MD_NONE )
+        return ( -1 );
```

Comment:
@catenacyber Could you fix the spacing here and below? It should be `return( -1 )`.